scripts/newleetcode.js

```diff
@@ -0,0 +1,535 @@
+/* Enum for languages supported by LeetCode. */
+const programmingLanguages = {
+  Python: '.py',
```

Comment:
what are the changes done in this js file?

Reply:
These were basically DOM manipulation changes that I fixed with the new UI, but since i had to run it parallel with the old UI, i made a new file called `newleetcode.js`, which had the same boilerplate as `leetcode.js` , the old UI file, but with different methods and member functions. There is one major problem as discussed in the PR, which if someone helps me with, I can fix it.  I have found the fix for difficulty tag of the problem and this can be migrated to v3 in the next PR, but for now, i want to address the 2nd issue. 
Feel free to merge this in the `dev` branch, and later we can merge it to `main`

Reply:
hey @sandeepk97 what do u think about merging these changes?